README.md

```diff
@@ -89,3 +105,15 @@ pest is the youngest among them, but is continuously improving.
 
 * [Packrat parsing](issues/7)
 * [parsing parallelization](issues/25)
+
+## Build & test locally
+Pest uses normal `Cargo` commands so you can build it by calling `cargo build` and run the test by calling `cargo test`.
+
+When you run the tests, you will see test names like `_0`, `_1` etc. Those are docstring tests.
+
+You can run the calculator example by using `cargo run --example calculator`.
+
+## FAQ
+
+### I get recursion errors, what should I do?
+You will need to increase the recursion limit of the crate by adding `#![recursion_limit = "x"]` in your `lib.rs` or `main.rs`. Try different values for `x` until it doesn't error anymore. You can start with a value of 200 and go up or down.
```

Comment:
Sorry I don't see what you mean, i don't see any extra space. I'll let you do the change